ethereum/oracles/src/noir/circuit/verifierData.ts

```diff
@@ -0,0 +1,29 @@
+import { Abi, InputMap, abiEncode } from '@noir-lang/noirc_abi';
+import { readTomlObject } from '../../util/file.js';
+import { Hex } from 'viem';
+import { CircuitAbi } from './abi.js';
+
+export class VerifierData {
```

Comment:
It seems to me you have two classes here:
`Abi`, with methods like `publicInputs`, `returnValues` and
`InputMap` with method`encodeSubset`.
So it seems to me we should have two classes each extending respective noir class: `VlayerAbi` and `VlayerInputMap`. 
Alternatively, we could create `AbiUtils` and `InputDataUtils`.
One more option is to extend existing prototypes, but that seems to be anti-pattern to me.